.github/workflows/pr.yaml

```diff
@@ -0,0 +1,117 @@
+name: PR checks
+on: pull_request
+env:
+  SECRET_KEY: insecure_test_key
+
+jobs:
+  run-image:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build image
+        run: DOCKER_BUILDKIT=1 docker build --tag jaas-ai .
+
+      - name: Run image
+        run: |
+          docker run --detach --network host --env SECRET_KEY=insecure_secret_key jaas-ai
+          sleep 1
+          curl --head --fail --retry-delay 1 --retry 30 --retry-connrefused http://localhost
+
+  run-dotrun:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dotrun
+        run: |
+          sudo snap install dotrun
```

Comment:
This project still uses the `./run` script

Reply:
is there anything stopping you from using `dotrun`?

Reply:
@barrymcgee has some issues with multipass & dotrun on his machine but as long as we maintain the ability to run the project without any run script that shouldn't be an issue.

However if you're going to keep this as using dotrun then can you also address the change from `./run` to `dotrun` as a followup?

Reply:
This one I think it's totally preferential, we are running `dotrun` or `./run` in local, so you might as well do it locally? But I can't get it to work with `./entrypoint` that's the only issue